app/components/application_record_card/view.rb

```diff
@@ -27,9 +27,9 @@ def subject
     end
 
     def route
-      return "No route provided" if record.record_type.blank?
+      return "No route provided" if record.training_route.blank?
 
-      record.record_type.humanize
+      t("activerecord.attributes.trainee.training_routes." + record.training_route)
```

Comment:
```suggestion
`t("activerecord.attributes.trainee.training_routes.#{record.training_route}")`
```

Reply:
whats with the "`" backticks?

Reply:
FYI there's some weirdness occuring as Github tries to display the suggestions. Here it is for reference: 

<img width="935" alt="Screenshot 2021-03-02 at 14 20 14" src="https://user-images.githubusercontent.com/616080/109661942-69626680-7b62-11eb-9e52-adc348512fa4.png">